code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -1849,6 +1849,16 @@
 	name = "Angeler"
 	icon_state = "angler"
 
+/datum/sprite_accessory/horns/broken
```

Comment:
put a space in between datum entries please

Reply:
Sure! I blame VScode for making me think there WAS a space..